### ISSUE TITLE
Add Student Contact Information in Student Profile

### ISSUE THREAD

**Issue by snoopyuri** [studentinsights/studentinsights#1087]
Adding phone number, email address, and address in student profile.  However, worried about things getting too crowded.  Possibility: when you hover over the name of the student, it pops up with this contact information.  If teacher wants to click on parent email address, it'll open outlook and they can send an email to parent.

**Comment by alexsoble** [comment]
Notes from call this morning with @snoopyuri and @gabivoicu:

+ Data-wise, what we want is:
  + address (already in database)
  + parent primary phone # (need to add)
  + parent primary email address (need to add)

+ Design goals for "contact" button and content: 
    - easily discoverable
    - minimal clutter added
    - easy to copy/paste content
    - outlook integration (nice to have)

+ Ways to implement design goals:
  + Add a button with an address book icon or the word "contact" near the top of the student profile
  + Clicking the button opens a modal or un-hides a div with the contact information, easily copy/pastable

+ Next steps
  + @gabivoicu makes a super-rough visual sketch (could be with a drawing tool, nothing fancy)
  + @alexsoble imports the contact information we don't have yet in the database

**Comment by alexsoble** [comment]
Hey there @gabivoicu! This PR (#1113) added some database fields and fake data that you can work with in your local environment to get started with the contact information UI.

Running `rake db:reset` will migrate the database and set up the seed data with fake phone numbers / emails so you can get started. 

I can work on pulling in the real data in parallel so that we're not blocking each other. Let me know if you have questions or see another opportunity for me to unblock you!

**Comment by gabivoicu** [comment]
@snoopyuri @alexsoble 

Here is my (pretty basic) mockups, they're not particularly well aligned, but they're a rough idea of what it would look like. 

What do you think about the modal open one? I am thinking of adding icons in front of each type of contact info, do you think that would be useful? (So I would add a phone icon in front of the phone number, etc).

Also, any new thoughts on using an address book icon in the first mockup vs the text `Contact`?

![icon](https://user-images.githubusercontent.com/8533377/31105103-34d39444-a7a7-11e7-9a7c-103d16bb956e.png)
![modal_open](https://user-images.githubusercontent.com/8533377/31105105-39716440-a7a7-11e7-95c4-bbb4394e8254.png)

**Comment by snoopyuri** [comment]
Really like the icon on the student profile.  Nice and clean.  The popup box gives us the info we need.  Not sure either about how to make the design of the popup box match with the existing style of Insights. 

Thanks, Gabi!

**Comment by gabivoicu** [comment]
@snoopyuri @alexsoble 
Are the places in the app where we use modals already? I try finding one last night, but couldn't after a quick search. If there's already one, I could imitate that and it would give it a more consistent feel.

**Comment by alexsoble** [comment]
@gabivoicu Try clicking the pale blue text next to Notes and Services that
says "(what is this?)"

**Comment by gabivoicu** [comment]
@alexsoble I will try that, thank you!

**Comment by gabivoicu** [comment]
@alexsoble I worked on this today and wanted to open a WIP PR but it appears I don't have permission to:
```
git push origin issue-1087-student-contact-info
remote: Permission to studentinsights/studentinsights.git denied to gabivoicu.
fatal: unable to access 'https://github.com/studentinsights/studentinsights.git/': The requested URL returned error: 403
```

I'll give more of an update when I open the PR, but I got the icon on the page, and clicking it open a modal 🎉

**Comment by jhilde** [comment]
@gabivoicu Looks like some great work! You can push to a branch in your fork of studentinsights/studentinsights and then open a PR against that branch in studentinsights/studentisights.

**Comment by gabivoicu** [comment]
Thank you, @jhilde, I will open a PR like that then.

For some timeline updates, I am planning to open a WIP PR Thursday or Friday, and then by the end of this week/early next week to have a reviewable PR with all of the work.

**Comment by alexsoble** [comment]
Closed by @gabivoicu! Great work!
